src/containers/stats/courttype/CourtTypeSagas.js

```diff
@@ -917,8 +910,13 @@ function* getMonthlyParticipantsByCourtTypeWorker(action :SequenceAction) :Gener
       );
       if (response.error) throw response.error;
       const monthlyParticipantsWithNoCheckInsByCourtType :Map = response.data;
+
       monthlyParticipantsByCourtType = monthlyParticipantsWithNoCheckInsByCourtType
-        .mergeDeep(monthlyParticipantsByCourtType);
+        .mergeDeepWith((oldVal, newVal) => oldVal + newVal, monthlyParticipantsByCourtType);
+
+      monthlyParticipantsByCourtType = monthlyParticipantsByCourtType
+        .map((courtTypeMap :Map) => courtTypeMap.keySeq().toList()
+          .map((personName :string) => fromJS({ personName, hours: courtTypeMap.get(personName) })));
```

Comment:
I think this is much more readable:

```suggestion
      monthlyParticipantsByCourtType = monthlyParticipantsByCourtType
        .map((courtTypeMap :Map) => (
          courtTypeMap
            .keySeq()
            .toList()
            .map((personName :string) => fromJS({ personName, hours: courtTypeMap.get(personName) }))
        ));
```

Reply:
ooh ya that's pretty—i'll update